cmc-domain-model/src/test/java/uk/gov/hmcts/cmc/domain/models/offers/SettlementTest.java

```diff
@@ -123,4 +124,30 @@ public void partyIsNotAllowedToRejectOfferWhenTheOnlyOfferWasMadeByThemselves()
     public void getLastStatementShouldThrowIllegalStateWhenNoStatementsHaveBeenMade() {
         settlement.getLastStatement();
     }
+
+    @Test
+    public void getLastOfferStatementShouldGiveLastStatementThatIsAnOffer() {
+        settlement.makeOffer(offer, MadeBy.CLAIMANT);
+        settlement.reject(MadeBy.DEFENDANT);
+
+        Offer counterOffer = SampleOffer.builder()
+            .withContent(COUNTER_OFFER)
+            .build();
+
+        settlement.makeOffer(counterOffer, MadeBy.DEFENDANT);
```

Comment:
Is defendant making counter offer? Shouldn't that be claimant?

Reply:
Based on the frontend yes, but the backend allows both way.